docs/sources/flow/setup/start-agent.md

```diff
@@ -155,8 +155,8 @@ AGENT_MODE=flow BINARY_PATH run CONFIG_PATH
 
 Replace the following:
 
-* `BINARY_PATH`: The path and Grafana Agent binary filename.
-* `CONFIG_PATH`: The path and Grafana Agent configuration filename.
+* `BINARY_PATH`: The path to the Grafana Agent binary.
+* `CONFIG_PATH`: The path to the Grafana Agent configuration.
```

Comment:
The original text included "file" to be more explicit. I think that is preferable but not strictly necessary.

https://github.com/grafana/agent/pull/5440/files#diff-7e58462b52df129cdaaeb3329af94622456fe4e4a2600d33f489f920ebeca55aL159

Reply:
Fixed